_posts/en/newsletters/2025-01-17-newsletter.md

```diff
@@ -0,0 +1,159 @@
+
+  Caseria also noted that existing mining pools find it financially
+  challenging to support the _full paid per share_ ([FPPS][topic fpps])
+  scheme where a miner is paid proportional to the entire block reward
+  (subsidy plus transaction fees) when they create a share.  He didn't
+  elaborate, but we understand the problem to be variance in fees
+  forcing pools to keep large reserves.  For example, if a pool miner
+  controls 1% of hashrate and creates shares on a template with about
+  1,000 BTC in fees and 3 BTC in subsidy, they would be owed by their
+  pool about 10 BTC.  However, if the pool doesn't mine that block and,
+  when they do mine a block, fees are back down to a fraction of the
+  block reward, the pool might only have 3 BTC total to split between
+  all of its miners, forcing it to pay from its reserves.  If that
+  happens too many times, the pool's reserves will be exhausted and it
+  will go out of business.  Pools address this in various ways,
+  including using [proxies for actual fees][news304 fpps proxy].
+
+  Developer vnprc [described][vnprc ehash] the solution he's been
+  [building][hashpool] that focuses on ecash shares received in the
+  PPLNS payout scheme.  He thinks this could be especially useful for
+  launching new pools: right now, the first miner to join a pool suffers
+  the same high variance as solo mining, so typically the only people
+  who can start a pool are existing large miners or those willing to
+  rent significant hashrate.  However, with PPLNS ecash shares, vnprc
+  thinks a pool could launch as a client of a larger pool, so even the
+  first miner to join the new pool would receive lower variance than
+  solo mining.  The intermediate pool could then sell the ecash shares
+  it earns to finance whatever payout scheme it chooses to pay
+  its miners.  Once the intermediate pool acquired a
+  significant amount of hashrate, it would also have leverage for
+  negotiating with larger pools about creating alternative block
+  templates that suit its miners.
+
+- **Offchain DLCs:** developer conduition [posted][conduition offchain]
+  to the DLC-dev mailing list about a contract protocol that allows an
+  offchain spend of the funding transaction signed by both parties to
+  create multiple [DLCs][topic dlc].  After the offchain DLC has settled
+  (e.g., all required oracle signatures have been obtained), a new
+  offchain spend can be signed by both parties to reallocate funds
+  according to the contract resolution.  A third alternative spend can
+  then allocate the funds to new DLCs.
+
+  Replies by Kulpreet Singh and Philipp Hoenisch linked to previous
+  research and development of this basic idea, including approaches that
+  allow the same pool of funds to be used for both offchain DLCs and
+  LN (see Newsletters [#174][news174 dlc-ln] and [#260][news260 dlc]).
+  A [reply][conduition offchain2] from conduition described his
+  proposal's major difference from previous proposals.
```

Comment:
Note that our summary doesn't go into any detail about what the major difference is.  I tried for over half an hour to understand it, and maybe I kinda got a glimmer of it, but I don't feel confident enough to try to summarize it at this late moment.  We'll have to trust that anyone who wants to learn more will follow the link.